django/test/utils.py

```diff
@@ -640,3 +640,14 @@ def setUp(self):
 
     def tearDown(self):
         self.logger.handlers[0].stream = self.old_stream
+
+
+def tag(*tags):
+    """
+    Decorator to add tags to test
+    """
+    def decorator(function):
```

Comment:
Will it work for whole classes? According to code it will, but in that case, may be it will be better to rename argument?